Src/NHibernate.Envers.Tests/NHibernate.Envers.Tests.csproj

```diff
@@ -7,6 +7,7 @@
 
   <ItemGroup>
     <EmbeddedResource Include="**/*.hbm.xml" Exclude="bin/**" />
+    <None Remove="Integration\Inheritance\TablePerClass\Discriminate\Mapping.hbm.xml" />

```

Comment:
This change look strange?

Reply:
I will fix this. I copied the hbm file from another test, so might have happened when I copied it over.